packages/mds-ingest-service/repository/entities/event-entity.ts

```diff
@@ -28,6 +30,7 @@ export interface EventEntityModel extends IdentityColumn, RecordedColumn {
   telemetry_timestamp: EventDomainModel['telemetry_timestamp']
   trip_id: EventDomainModel['trip_id']
   service_area_id: EventDomainModel['service_area_id']
+  telemetry?: Nullable<TelemetryEntityModel>
```

Comment:
Hmm including this on the entity model seems a little odd, cause we don't _actually_ have that persisted as a column on the event entity. Curious what @drtyh2o's thoughts are

Reply:
I checked w/ @drtyh2o ! This is _how_ TypeOrm maps a 2nd model onto the first, with `leftJoinAndMapOne()` 
Otherwise, that update to the mapper wouldn't have passed type checks.

Reply:
Ahh interesting, pretty cool!